modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -123,7 +122,7 @@ If the decryption process fails, the Mule environment displays an error:
 Client ID cannot be decrypted
 ----
 
-If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform. Subsequently you will not be able to download policies or contracts.
+If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform. Subsequently you will not be able to download policies or contracts, or generate analytics information for tracked APIs.
```

Comment:
```suggestion
If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform, which means that you cannot download policies or contracts or generate analytics information for tracked APIs.
```

Reply:
Kept as is because I feel when the sentence is too long, comprehension is impaired.